utils/type_checker.py

```
@@ -0,0 +1,21 @@
+
+class c:
+    red = '\033[91m'
+    end = '\033[0m'
+
+with open('public/static/data/games/cscd.txt', 'r') as f:
+    for line in f:
+        line = ast.literal_eval(line)
+        first = line["name"].split(" ")[0]
+        try:
+            second = line["type"]
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)